ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -260,10 +260,12 @@ def _router_bgp_vlans(self) -> dict | None:
 
         return None
 
-    def _router_bgp_vlans_vlan(self, vlan, tenant) -> dict | None:
+    def _router_bgp_vlans_vlan(self, vlan, tenant, vrf=None) -> dict | None:
         """
         Return structured config for one given vlan under router_bgp.vlans
         """
+        if vrf is None:
```

Comment:
instead of having None as default value and then setting {}, just add {} as the `vrf` paremeter when you configure l2vlans.